package.json

```diff
@@ -2,56 +2,80 @@
   "name": "express-api-skeleton",
   "version": "1.0.0",
   "description": "Skeleton for Express APIs.",
-  "main": "app.js",
-  "scripts": {
-    "start": "./node_modules/.bin/gulp run",
-    "lint": "./node_modules/.bin/gulp lint",
-    "test": "./node_modules/.bin/gulp test"
+  "homepage": "https://github.com/osu-mist/express-api-skeleton#readme",
+  "bugs": {
+    "url": "https://github.com/osu-mist/express-api-skeleton/issues"
   },
-  "pre-commit": [
-    "lint"
-  ],
   "repository": {
     "type": "git",
     "url": "git+https://github.com/osu-mist/express-api-skeleton.git"
   },
-  "author": "osu-mist",
   "license": "AGPL-3.0",
-  "bugs": {
-    "url": "https://github.com/osu-mist/express-api-skeleton/issues"
+  "author": "osu-mist",
+  "main": "dist/app.js",
+  "scripts": {
+    "babel": "./node_modules/.bin/gulp babel",
+    "lint": "./node_modules/.bin/gulp lint",
+    "start": "./node_modules/.bin/forever dist/app.js",
+    "test": "./node_modules/.bin/gulp test",
+    "typecheck": "./node_modules/.bin/gulp typecheck"
```

Comment:
Could you also add a `build` script that calls the `build` gulp task? Then, this script can be used by the build Dockerfile